docs/source/daml/reference/data-types.rst

```diff
@@ -33,7 +33,11 @@ Table of built-in primitive types
    * - ``Numeric n``
      - fixed point decimal numbers
      - ``1.0``
-     - `Numeric n` values are rational numbers with up to ``38`` digits. The scale parameter ``n`` controls the number of digits after the decimal point, so for example, ``Numeric 10`` values have 10 decimal places, and ``Numeric 20`` values have 20 decimal places. The value of ``n`` must be between ``0`` and ``37`` inclusive.
+     - ``Numeric n`` values are rational numbers with up to ``38`` decimal digits. The scale parameter ``n`` controls the number of digits after the decimal point, so for example, ``Numeric 10`` values have 10 decimal places, and ``Numeric 20`` values have 20 decimal places. The value of ``n`` must be between ``0`` and ``37`` inclusive.
```

Comment:
hm, I view Numeric as always having 38 decimal digits including trailing zeroes.

Reply:
How's this? https://github.com/digital-asset/daml/pull/9336/commits/ca9a518c3879ec2f9e22bfcd1cf42debf8061601

Reply:
I like it, thanks!